src/dts/logger.d.ts

```diff
@@ -19,6 +19,7 @@ export declare enum LogLevel {
 export declare class ComponentLogger implements ZLUX.ComponentLogger {
     private parentLogger;
     private componentName;
+    MESSAGES: any;
```

Comment:
private messages or private _messages would be more appropriate.
ALL_CAPS are for things that are immutable or intended to be.

Reply:
Fixed, makes sense